jaeger/src/main/java/io/opentelemetry/example/jaeger/ExampleConfiguration.java

```diff
@@ -41,7 +42,7 @@ static OpenTelemetry initOpenTelemetry(String jaegerEndpoint) {
     // Set to process the spans by the Jaeger Exporter
     SdkTracerProvider tracerProvider =
         SdkTracerProvider.builder()
-            .addSpanProcessor(SimpleSpanProcessor.create(jaegerExporter))
+            .addSpanProcessor(BatchSpanProcessor.builder(jaegerOtlpExporter).build())
```

Comment:
This feels like a good idea because most users want this...but it wasn't entirely necessary was it?

Reply:
No, not necessary! Updated because not good to have examples doing things we would advise against.